### PR TITLE
Fixes #15042: Updating changes/repairs (probably) uses too much memory 

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15042

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala [hunk 1]
```diff
@@ -195,18 +193,19 @@ class ReportsExecutionService (
     Future {
       //update changes by rules
       (for {
-        changes <- reportsRepository.getChangeReportsOnInterval(lowestId, highestId)
-        updated <- cachedChanges.update(changes)
+        changes         <- reportsRepository.getChangeReportsOnInterval(lowestId, highestId)
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala]: as an afterthought, it should probably be the getChangeReportsOnInterval which ought to have a different query

Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala]: Afterthought of #15041?

Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala]: OK, after some more hours on the code, I understand that you want to change the query to only return the relevant part (ruleId, executionTimestamp). That seems to be a much better solution to minimize memory, I agree!

**Comment by fanf** [comment]
Superseeded by https://github.com/Normation/rudder/pull/2281

**Pull request closed without merging**